src/main/java/Duke/DukeTasks/Task.java

```diff
@@ -0,0 +1,64 @@
+package Duke.DukeTasks;
+
+import java.time.LocalDate;
+
+public class Task {
+    private String name;
+    private boolean finished = false;
```

Comment:
Maybe you could change this to the name of this boolean
```suggestion
    private boolean isFinished = false;
```